test/e2e_node/swap_test.go

```diff
@@ -0,0 +1,254 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2enode
+
+import (
+	"context"
+	"fmt"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/features"
+	"k8s.io/kubernetes/pkg/kubelet/types"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	testutils "k8s.io/kubernetes/test/utils"
+	admissionapi "k8s.io/pod-security-admission/api"
+	"path/filepath"
+	"strconv"
+)
+
+const (
+	cgroupBasePath        = "/sys/fs/cgroup/"
+	cgroupV1SwapLimitFile = "/memory/memory.memsw.limit_in_bytes"
+	cgroupV2SwapLimitFile = "memory.swap.max"
+	cgroupV1MemLimitFile  = "/memory/memory.limit_in_bytes"
+)
+
+var _ = SIGDescribe("Swap [NodeConformance]", func() {
```

Comment:
is there CI job that run it? I want to confirm that these tests are green

Reply:
Yes, it runs (and passed) on many Ci jobs.
Some examples:
* [pull-kubernetes-node-e2e-containerd](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/118764/pull-kubernetes-node-e2e-containerd/1679161050328469504)
* [pull-kubernetes-node-crio-cgrpv2-e2e](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/118764/pull-kubernetes-node-crio-cgrpv2-e2e/1679106741469450240)
* [pull-kubernetes-node-swap-fedora](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/118764/pull-kubernetes-node-swap-fedora/1678023403514105856)